.gitignore

```diff
@@ -36,8 +36,8 @@ node_modules
 .node_repl_history
 
 # Transpiled code
-lib
-packages/mongodb-memory-server-core/lib
+#lib
+#packages/mongodb-memory-server-core/lib
```

Comment:
Why is this removed and all the generated files included?

Reply:
I am testing in the CI/CD environment with the forked version for which I needed to publish to npm.